packages/labs/package.json

```diff
@@ -9,7 +9,9 @@
   "dependencies": {
     "@blueprintjs/core": "^1.23.0",
     "classnames": "^2.2",
-    "pure-render-decorator": "1.2.1",
+    "popper.js": "1.11.0",
```

Comment:
since this is labs, I'd say this is ok for now. but we really gotta fix the typings soon